Assets/MRTK/SDK/Features/UX/Scripts/BoundsControl/BoundsControl.cs

```diff
@@ -564,6 +564,21 @@ public ElasticsManager ElasticsManager
         private Vector3[] boundsCorners = new Vector3[8];
         public Vector3[] BoundsCorners { get; private set; }
 
+        // Current actual flattening axis, derived from FlattenAuto, if set
+        private FlattenModeType actualFlattenAxis {
```

Comment:
nit on the casing here: we've tended to use PascalCase for all properties, regardless of their access modifiers.

https://docs.microsoft.com/en-us/windows/mixed-reality/mrtk-unity/contributing/coding-guidelines?view=mrtkunity-2021-05#naming-conventions

Reply:
Thanks, fixed.